src/components/Recipe/RecipeItem/RecipeItem.tsx

```diff
@@ -74,4 +86,76 @@ const RecipeItem = ({ recipe, isMemberPage = false }: RecipeItemProps) => {
   );
 };
 
-export default memo(RecipeItem);
+const ProductCircleButton = () => {
+  const { recipe } = useRecipeItemValueContext();
+  const { products } = recipe;
+
+  return (
+    <ul className={productCircleWrapper}>
+      {displaySlice(true, products, 3).map(({ id, image }, idx) => (
+        <li key={id} className={productCircleListWrapper}>
+          <img
+            src={image}
+            alt="사용한 상품"
+            className={products.length > 3 && idx === 2 ? thirdProductImage : productImage}
+          />
+          {idx === 2 && (
+            <Text size="caption3" weight="medium" className={recipeProductsCount}>
+              +{products.length - 3}
+            </Text>
+          )}
+        </li>
+      ))}
+    </ul>
+  );
+};
+
+const Title = () => {
+  const { recipe } = useRecipeItemValueContext();
+  const { title } = recipe;
```

Comment:
```suggestion
  const { recipe: { title } } = useRecipeItemValueContext();
```

너무 반복된다면 이렇게 할 수 있었던거 같은데 잘 기억이 안 나네요 ㅎㅎ